app/templates/event/partials/edit_event_details_tab.html

```diff
@@ -40,15 +59,37 @@
     </select>
   </div>
 
+  <div class="form_row_end">
+      <select name="availability">
+          <option value="False">Free</option>
+          <option value="True" selected>Busy</option>
+      </select>
+      <select name="privacy">
+          <option value="private">Private</option>
+          <option value="public" selected>Public</option>
+      </select>
+  </div>
+</div>
 
-    <div class="form_row_end">
-        <select name="availability">
-            <option value="False">Free</option>
-            <option value="True" selected>Busy</option>
-        </select>
-        <select name="privacy">
-            <option value="private">Private</option>
-            <option value="public" selected>Public</option>
-        </select>
-    </div>
-</div>
\ No newline at end of file
+<script>
+  var items_num = 1;
+
+  function addItem() {
+    var item = document.getElementById("Item").cloneNode(true);
+    item.style.display = "flex";
+    item.childNodes[7].id += String(items_num);
+    item.id = item.id + String(items_num);
+    items_num++;
+    for (i = 0; i < item.children.length; i++ ) {
+      if (item.children[i].tagName == 'INPUT') {
+        item.children[i].setAttribute('required', 'required');
+      }
+    }
+    document.getElementById("Items").appendChild(item);
+  }
+
+  function removeItem(clicked_id) {
+    document.getElementById(clicked_id).parentElement.remove();
+    items_num--
+  }
+</script>
```

Comment:
Put scripts in a dedicated file.

Reply:
The script is now located in static/js/shared_list.js